src/DiffImages.jl

```diff
@@ -1,16 +1,21 @@
 module DiffImages
-
 using Flux,
-      Images,
+      ImageCore,
       Zygote,
       ChainRules,
-      ImageTransformations
+      ImageTransformations,
+      StaticArrays,
+      CoordinateTransformations,
+      ColorVectorSpace,
+      Interpolations,
+      ChainRulesCore
 
-using Flux:@adjoint, @functor, unsqueeze
-using ChainRules:NoTangent
+using Flux: @functor, unsqueeze
+using Zygote: @adjoint
+using ChainRulesCore: NoTangent
 
 export colorify, channelify
 include("colors/conversions.jl")
-include("geometry/warp.jl")
+include("geometry/adjoints.jl")
 
-end # module
+end
```

Comment:
Check your editor settings please, this should be in the Linux file endings

Reply:
Oh, so should I revert the changes? Should I keep `# module`?

Reply:
Do you see the little red circle with a cross in the diff? That's what I was referring to. No harm keeping the comment, no hard removing. File endings should be neat though.

Reply:
I had to go to the diff to check out the red cross, did not know about this. My editor defaults to clrf I think so. Should I insert newlines at the end of files?

Reply:
Your editor must have some settings to control this, change that, save the files and commit

Reply:
Done.